cmd/image-builder/main.go

```diff
@@ -154,7 +154,8 @@ func main() {
 			if values.Error != nil {
 				fields["error"] = values.Error
 			}
-			logrus.WithFields(fields).Infof("Processed request %s %s", values.Method, values.URI)
+			logrus.WithContext(c.Request().Context()).
+				WithFields(fields).Infof("Processed request %s %s", values.Method, values.URI)
```

Comment:
Here `WithContext` was missing, somewhat unrelated to the change but I think it is worth fixing.

Reply:
do you think also adding
`"path":   c.Path(),`
to the fields here would be beneficial as this is better for log entry searches/grouping (in contrast to `values.URI`) I guess
but no stopper for this PR I guess